narwhal/primary/src/proposer.rs

```diff
@@ -410,6 +410,15 @@ impl Proposer {
         debug!("Dag starting at round {}", self.round);
         let mut advance = true;
 
+        let position = self
+            .committee
+            .authorities()
+            .position(|(name, _a)| name.clone() == self.name)
+            .unwrap();
+        if position == 0 {
+            debug!("I have been selected as a low performing authority");
```

Comment:
Why is it a low performing authority if its position is 0?

Reply:
This is for labnet testing, and was intended as a temporary thing